packages/remote-config/src/api.ts

```diff
@@ -111,7 +111,11 @@ export async function activate(remoteConfig: RemoteConfig): Promise<boolean> {
     // config.
     return false;
   }
-  const experiment = new Experiment(rc._storage, rc._analyticsProvider);
+  const experiment = new Experiment(
+    rc._storage,
```

Comment:
Suggestion: why not just pass `rc` as an argument? Then assign all the properties inside the Experiment constructor.

Reply:
Done